src/main/java/Check.java

```diff
@@ -0,0 +1,72 @@
+public class Check {
+     //Function to check if an input has mark as the first word
+     public static boolean isMark(String input){
+        if(input.length()>=4){
+            if("mark".equalsIgnoreCase(input.substring(0,4))){
+                return true;
+            }
+        }
+        return false;
+    }
+
+    //Function to check if an input has unmark as the first word
+    public static boolean isUnmark(String input){
+        if(input.length()>=6){
+            if("unmark".equalsIgnoreCase(input.substring(0,6))){
+                return true;
+            }
+        }
+        return false;
```

Comment:
```suggestion
        bool isValidLen = (input.length()>=6);
        return (isValidLen && "unmark".equalsIgnoreCase(input.substring(0,6)))
```
Maybe consider not having nested conditional if not necessary